File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Cpls/CPL0000Q00SelectOUT0101ListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Cpls
{
    [Serializable]
	public class CPL0000Q00SelectOUT0101ListItemInfo
	{
		private String _bunho;
		private String _suname;
		private String _birth;
		private String _sex;
		private String _age;

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String Suname
		{
			get { return this._suname; }
			set { this._suname = value; }
		}

		public String Birth
		{
			get { return this._birth; }
			set { this._birth = value; }
		}

		public String Sex
		{
			get { return this._sex; }
			set { this._sex = value; }
		}

		public String Age
		{
			get { return this._age; }
			set { this._age = value; }
		}

		public CPL0000Q00SelectOUT0101ListItemInfo() { }

		public CPL0000Q00SelectOUT0101ListItemInfo(String bunho, String suname, String birth, String sex, String age)
		{
			this._bunho = bunho;
			this._suname = suname;
			this._birth = birth;
			this._sex = sex;
			this._age = age;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/cpls/CPL0000Q00SelectOUT0101ListItemInfo.java
package nta.med.data.model.ihis.cpls;

public class CPL0000Q00SelectOUT0101ListItemInfo {
	private String bunho;
    private String suname;
    private String birth;
    private String sex;
    private Integer age;
	public CPL0000Q00SelectOUT0101ListItemInfo(String bunho, String suname,
			String birth, String sex, Integer age) {
		super();
		this.bunho = bunho;
		this.suname = suname;
		this.birth = birth;
		this.sex = sex;
		this.age = age;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getBirth() {
		return birth;
	}
	public void setBirth(String birth) {
		this.birth = birth;
	}
	public String getSex() {
		return sex;
	}
	public void setSex(String sex) {
		this.sex = sex;
	}
	public Integer getAge() {
		return age;
	}
	public void setAge(Integer age) {
		this.age = age;
	}
}
